<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RGScanDialog</class>
 <widget class="QDialog" name="RGScanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplayGain Scanner</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Progress</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Track Gain</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Album Gain</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Track Peak</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Album Peak</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="skipScannedCheckBox">
       <property name="text">
        <string>Skip already scanned files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="trackCheckBox">
       <property name="text">
        <string>Write track gain/peak</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="albumCheckBox">
       <property name="text">
        <string>Write album gain/peak</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="calculateButton">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="writeButton">
     <property name="text">
      <string>Write Tags</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RGScanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RGScanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
